CONTRIBUTING.md

```diff
@@ -74,6 +74,7 @@ again.
 ### Issue management
 
 * We use [Zenhub](https://www.zenhub.com) to manage the queue, in addition to what we do with labels.
+  * You will need to install ZenHub extension to your browser to show the board. Please see [ZenHub Browser Extension](https://www.zenhub.com/extension) for detail.
```

Comment:
i think we can shorten this even a little more by dropping the second sentence, and putting the https://www.zenhub.com/extension link on the words "ZenHub extension" in the first sentence.

Reply:
:+1: Seems better. I updated.